packages/grafana-perf-tests/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "typescript",
+  "version": "1.0.0",
+  "main": "index.js",
+  "devDependencies": {
+    "@babel/core": "7.13.16",
+    "@babel/plugin-proposal-class-properties": "7.13.0",
+    "@babel/plugin-proposal-object-rest-spread": "7.13.8",
+    "@babel/preset-env": "7.13.15",
+    "@babel/preset-typescript": "7.13.0",
+    "@faker-js/faker": "7.6.0",
+    "@types/k6": "~0.39.0",
+    "@types/shortid": "0.0.29",
+    "@types/webpack": "5.28.0",
+    "babel-loader": "8.2.2",
+    "clean-webpack-plugin": "4.0.0-alpha.0",
+    "copy-webpack-plugin": "^9.0.1",
+    "cross-env": "7.0.3",
+    "shortid": "2.2.16",
+    "ts-node": "10.9.1",
+    "typescript": "4.2.4",
+    "webpack": "5.35.1",
+    "webpack-cli": "4.6.0",
+    "webpack-glob-entries": "^1.0.1"
+  },
+  "scripts": {
+    "build": "webpack",
+    "prepare": "yarn run prepare:object-store-test",
```

Comment:
I believe Lerna will call this during the publish phase. See [npm docs](https://docs.npmjs.com/cli/v8/using-npm/scripts#prepare-and-prepublish) for reference. Is that intended?